lib/developer_portal/app/controllers/developer_portal/admin/messages/outbox_controller.rb

```diff
@@ -50,6 +50,10 @@ def create
   private
 
   def build_message
-    @message = current_account.messages.build((params[:message] || {}).merge(:origin => "web"))
+    @message = current_account.messages.build((message_params[:message] || {}).merge(:origin => "web"))
+  end
+
+  def message_params
+    params.permit!.to_h
```

Comment:
In most cases yes, but when you can't tell the exact parameters it's better to allow all instead of guessing and introducing a bug.

At least that's my point of view because I tried to find the exact parameters but I wasn't successful. Also Developer portal is really old.